### PR TITLE
grid layout for cv and about me / css structure with seperate file

### PR DESCRIPTION
**Pull request by Justus-Becker-Rabbenstein**

### CODE REVIEW

File: css/aboutmecv.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.aboutmecv {
+    border: var(--troubleshootingBorder);
+    display: grid;
+    grid-template-rows: 3;
+    grid-template-columns: 1;
+  }
+  .aboutmecv_aboutme {
```
Comment (actyralf) [inline css/aboutmecv.css]: I think you missed the second underscore for BEM notation: `aboutmecv__aboutme`

File: index.html [hunk 2]
```diff
@@ -12,6 +12,7 @@
       <link rel="stylesheet" href="css/imageandinterests.css" />
```
Comment (actyralf) [inline index.html]: This is fine but you should consider having an `index.css` that imports all stylesheets in the correct order.
Then you could link only to this one from your HTML. But this is definitely a matter of taste...

File: css/aboutmecv.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
+.aboutmecv {
```
Comment (actyralf) [inline css/aboutmecv.css]: can you think of a better name? `aboutmecv` is hard to read and understand... Isn't this just `cv` or `cv-card`?

**Review by actyralf — APPROVED** [review]
Hey Justus, well done!

I left a few small comments with improvement suggestions, but overall it's great work!

File: css/aboutmecv.css (hunk 1 above)
Comment (Justus-Becker-Rabbenstein) [reply css/aboutmecv.css]: Fixed.

File: css/aboutmecv.css (hunk 3 above)
Comment (Justus-Becker-Rabbenstein) [reply css/aboutmecv.css]: Fixed.

File: index.html (hunk 2 above)
Comment (Justus-Becker-Rabbenstein) [reply index.html]: Sorry, I don't know what an index.css is.

Comment (Justus-Becker-Rabbenstein) [reply index.html]: Found it in the course resources and fixed the layout accordingly.

**Pull request merged**